packages/swapper/src/swappers/baseErrors.ts

```diff
@@ -0,0 +1,10 @@
+export enum SWAPPER_ERRORS {
+  SLIPPAGE_EXCEEDED = 'trade.errors.slippageExceeded',
+  UNDER_MINIMUM_AMOUNT = 'trade.errors.underMinimumAmount',
+  CONSUMED_MORE_FEES = 'trade.errors.consumedMoreFees',
+}
+
+export type SwapperErrorMapping = {
+  key: SWAPPER_ERRORS
+  value: string
+}
```

Comment:
See https://github.com/shapeshift/web/pull/8433/files#r1894277094, we're effectively reinventing objects.

Reply:
Yeah this is just a `Record<SWAPPER_ERRORS, string>`.

Reply:
Changed in [17f168a](https://github.com/shapeshift/web/pull/8433/commits/17f168a1ba52efd3e9b3f5e55e1b8f1e2266db37)